jax/_src/numpy/lax_numpy.py

```diff
@@ -6375,7 +6375,10 @@ def diagflat(v: ArrayLike, k: int = 0) -> Array:
   return res
 
 
-def trim_zeros(filt, trim='fb'):
+deprecations.register("jax-numpy-trimzeros-lists")
+deprecations.register("jax-numpy-trimzeros-not-1d-array")
```

Comment:
Let's register these in `deprecations.py`, alongside other registered deprecations.

Reply:
Done